docs/contributors/issue-management.md

```diff
@@ -0,0 +1,406 @@
+# Issue Management
```

Comment:
Same here. Why repeat the same contents?

Reply:
Deleted. Thanks.